### ISSUE TITLE
Long load times on DataSet page when many datasets are present

### ISSUE THREAD

**Issue by kiivihal** [delving/narthex#92]
When 50+ datasets are present the initial load time of the dataset page becomes progressively slower. Once all are loaded you will get a reasonable responsiveness but it seems that there is a lot of spurious network traffic that puts unneeded burden on the Play appliciton and the Triple-Store.

**Comment by hanswesterbeek** [comment]
I don't see any evidence of 'spurious network traffic'. I just see a request to `/narthex/app/dataset-list`.

Please provide proof.

**Comment by hanswesterbeek** [comment]
I talked to @zemyatin and we agree the cause is probably that the `dataset-list` endpoint sends waaay to much data. Look at what it sends for a single dataset:

```json
{  
    "@id":"http://devorg.localhost/resource/dataset/dfvdfv",
    "@type":"http://schemas.delving.eu/narthex/terms/Dataset",
    "actorOwner":"http://devorg.localhost/resource/actor/admin",
    "datasetAggregator":"Rijksdienst voor het Cultureel Erfgoed",
    "datasetCharacter":"character-mapped",
    "datasetDescription":"",
    "datasetLanguage":"nl",
    "datasetMapToPrefix":"edm",
    "datasetName":"Collectie Schraven",
    "datasetOwner":"Collectie Schraven",
    "datasetRecordCount":"3",
    "datasetRights":"http://creativecommons.org/licenses/by-nc/3.0/nl/",
    "datasetSpec":"dfvdfv",
    "harvestDataset":"",
    "harvestPrefix":"",
    "harvestType":"pmh",
    "harvestURL":"",
    "orgId":"devorg",
    "publishIndex":"true",
    "publishLOD":"true",
    "publishOAIPMH":"true",
    "stateMappable":"2016-09-13T10:17:34+02:00",
    "stateProcessable":"2016-09-13T10:17:35+02:00",
    "stateSourced":"2016-09-13T10:17:34+02:00",
    "http://schemas.delving.eu/narthex/terms/synced":false,
    "@context":{  
      "datasetRights":"http://schemas.delving.eu/narthex/terms/datasetRights",
      "publishLOD":"http://schemas.delving.eu/narthex/terms/publishLOD",
      "datasetRecordCount":"http://schemas.delving.eu/narthex/terms/datasetRecordCount",
      "stateProcessable":"http://schemas.delving.eu/narthex/terms/stateProcessable",
      "publishOAIPMH":"http://schemas.delving.eu/narthex/terms/publishOAIPMH",
      "datasetAggregator":"http://schemas.delving.eu/narthex/terms/datasetAggregator",
      "datasetMapToPrefix":"http://schemas.delving.eu/narthex/terms/datasetMapToPrefix",
      "datasetLanguage":"http://schemas.delving.eu/narthex/terms/datasetLanguage",
      "datasetDescription":"http://schemas.delving.eu/narthex/terms/datasetDescription",
      "synced":{  
        "@id":"http://schemas.delving.eu/narthex/terms/synced",
        "@type":"http://www.w3.org/2001/XMLSchema#boolean"
      },
      "datasetName":"http://schemas.delving.eu/narthex/terms/datasetName",
      "orgId":"http://schemas.delving.eu/narthex/terms/orgId",
      "harvestURL":"http://schemas.delving.eu/narthex/terms/harvestURL",
      "harvestPrefix":"http://schemas.delving.eu/narthex/terms/harvestPrefix",
      "publishIndex":"http://schemas.delving.eu/narthex/terms/publishIndex",
      "actorOwner":{  
        "@id":"http://schemas.delving.eu/narthex/terms/actorOwner",
        "@type":"@id"
      },
      "datasetSpec":"http://schemas.delving.eu/narthex/terms/datasetSpec",
      "harvestDataset":"http://schemas.delving.eu/narthex/terms/harvestDataset",
      "stateSourced":"http://schemas.delving.eu/narthex/terms/stateSourced",
      "datasetCharacter":"http://schemas.delving.eu/narthex/terms/datasetCharacter",
      "datasetOwner":"http://schemas.delving.eu/narthex/terms/datasetOwner",
      "stateMappable":"http://schemas.delving.eu/narthex/terms/stateMappable",
      "harvestType":"http://schemas.delving.eu/narthex/terms/harvestType"
    }
  }
```

It is easy to imagine how this grows too quickly :)

A solution would be to:
 1. Send an abstract of the list (only the stuff that is /really/ used in the list-view
 2. Make a detail-request

Another (perhaps easier) option would be to force a gzipped-response. That will reduce the size of the response enormously.

**Comment by hanswesterbeek** [comment]
Fixed by commit 8d9f15314363685cb5
